sig-network/README.md

```diff
@@ -135,6 +148,31 @@ The following [subprojects][subproject-definition] are owned by sig-network:
 [subproject-definition]: https://github.com/kubernetes/community/blob/master/governance.md#subprojects
 [working-group-definition]: https://github.com/kubernetes/community/blob/master/governance.md#working-groups
 <!-- BEGIN CUSTOM CONTENT -->
+
+### Subproject Leads
+
+SIG network provides some additional responsibilities for subproject leads beyond was in covered in the [standard subproject definition][subprojects].
+Most of these additional responsibilities relate to communication about their projects within SIG Network, and to the greater community:
+
+- **Transparent Project Planning, Maintenance and Communication:**
+  - Subproject Leads **MUST** provide transparent view into their historical and future project plans (eg; using GitHub project boards, KEPs, or custom enhancement proposals (see [GEPs]/[NPEPs])).
+  - Subproject leads **MUST** create, maintain and be present in a public [Kubernetes Slack] channel with the naming `#sig-network-<subproject>`.
+  - Subproject leads **SHOULD** create and maintain a regular public [Zoom] sync on the [SIG Network Calendar].
+  - Subproject leads **MUST** keep the project in good health through regular issue triaging, PR reviews, CI health monitoring and checking testgrids (delegating this to other members in the community is good!)
+  - Projects that include the official CRDs using `k8s.io` must go through API review process as is described here https://github.com/kubernetes/community/blob/master/sig-architecture/api-review-process.md and Subproject leads are expected to make sure this well defined processes across Kubernetes is followed.
```

Comment:
Unfortunately "official CRDs" is only a colloquialism at the moment, I recommend this language instead:

```suggestion
  - Projects that distribute CRDs **MUST** go through the [API review process](https://github.com/kubernetes/community/blob/master/sig-architecture/api-review-process.md) for major releases.
```

Reply:
Indeed, it was meant as more of an extra reminder but if you feel its too redundant I'm fine with dropping it.

Reply:
this was suggested by @aojea as well from previous review iterations.. will wait to see what he thinks..

Reply:
I saw some confusion around this several times so I thought it was good to reinforce the message

Reply:
Dan as long as you're not strongly opposed, then I propose we leave this in as an extra reminder?

Reply:
I was chatting with @robscott about this recently, I think that we need to reword this as something 

> Subprojects should follow project guidelines and involve the SIG in user-facing changes, such as new APIs, which require API review.

Reply:
@thockin please chime in and see if we can get this merged

Reply:
Anything with a *k8s.io prefix MUST go thru API review.  Other sub-projects SHOULD.  If I recall the rules correctly.

Reply:
That being the case, my preference would be that we add an explicit MUST for any official SIG Network sub-project that distributes CRD APIs. This helps us to (at bare minimum) stay in touch with what the project is up to (albeit the other stuff here about checking in regularly should help with that as well). One of the high level goals of adding these additional responsibilities is to promote more communication.

Reply:
I am fine with that.

Reply:
tried to update it based on the discussion here, resolving this let's see if the next push makes it better.